x/gov/handler.go

```diff
@@ -595,22 +598,36 @@ func (h TextResolutionHandler) Check(ctx weave.Context, db weave.KVStore, tx wea
 }
 
 func (h TextResolutionHandler) Deliver(ctx weave.Context, db weave.KVStore, tx weave.Tx) (*weave.DeliverResult, error) {
-	_, err := h.validate(ctx, db, tx)
+	msg, err := h.validate(ctx, db, tx)
 	if err != nil {
 		return nil, err
 	}
-	// TODO: store this resolution somewhere
-	return &weave.DeliverResult{}, nil
+
+	proposal := CtxProposal(ctx)
+	if proposal == nil {
+		return nil, errors.Wrap(errors.ErrNotFound, "no proposal set for passed resolution")
+	}
+	resolution := &Resolution{
+		Metadata: msg.Metadata,
```

Comment:
I think you are right. I'm going to change that

Reply:
So, we should explicitly set it to 1?

I had that first, then figured the other approach was better.
Thank you for the feedback. It would be great if you could add some tips on usage in the `migrations` package. I am still at a loss sometimes on how to use it properly

Reply:
I have no tips, I think we are still figuring this out :see_no_evil: 

I don't have any better idea than hardcoding it to 1. This might require a manual change when upgraded. I think this is better than an automatic upgrade

Reply:
I read the migrations/doc.go and the empty Metadata, with schema to be filled by the Bucket approach was recommended.

I think we have too little experience to see if it is the *right* approach, but at least we can have a *consistent* approach, which makes future changes and testing easier